docs/ref/dotq.md

```diff
@@ -377,7 +377,7 @@ Where
 
 -   `d` is a directory handle
 -   `p` is a partition of a database
--   `f` a field of the table named by
+-   `f` a field of the table (required to be present in table since 4.1t 2021.09.03) named by
```

Comment:
Does this make sense? The table named by what/whom? Or is that just terminology?

Reply:
believe originally, its the 't' that follows on the next line that its referring to

Reply:
so can we say:
named by `t`, below